web/lib/types/my-plan.ts

```diff
@@ -60,6 +60,7 @@ export function deserializeMyPlan(serialized: Serialized): MyPlan {
 }
 
 export function serializeMyPlans(plans: MyPlan[]): Serialized {
+  console.log(plans)
```

Comment:
Odebrat než to můžeme mergenout

Reply:
Odebráno + odebrány další zapomenuté console logy